lib/factory_ex.ex

```diff
@@ -2,7 +2,16 @@ defmodule FactoryEx do
   @build_definition [
     keys: [
       type: {:in, [:atom, :string, :camel_string]},
-      doc: "Sets the type of keys to have in the built object, can be one of `:atom`, `:string` or `:camel_string`"
+      doc:
+        "Sets the type of keys to have in the built object, can be one of `:atom`, `:string` or `:camel_string`"
+    ],
+    relational: [
+      type: {:or, [{:list, :atom}, :keyword_list]},
+      doc: "Sets the ecto schema association fields to generate, can be a list of `:atom` or `:keyword_list`"
+    ],
+    check_owner_key?: [
```

Comment:
what's an owner key?

Reply:
That refers to the key on the `owner` schema used for an association. So if a `User` belonged to a team the belongs to association would have an `owner_key` of `team_id`. This has been removed however as It's not
needed. Ecto will raise an error if the owner key is specified along with parameters for the association.